pkg/internal/checkup/checkup.go

```diff
@@ -102,6 +102,10 @@ func (c *Checkup) Run(ctx context.Context) error {
 	}
 	c.results.VMUnderTestActualNodeName = c.vmi.Status.NodeName
 
+	if c.results.OslatMaxLatency > c.cfg.OslatLatencyThreshold {
```

Comment:
Please consider squashing this commit with the previous one.

Reply:
DONE